plugins/modules/zos_backup_restore.py

```diff
@@ -322,6 +323,12 @@
     backup_name: /tmp/temp_backup.dzp
     sms_storage_class: DB2SMS10
     sms_management_class: DB2SMS10
+
+- name: Restore data sets from backup stored with their original HLQ from when they were archived .
```

Comment:
`stored` seems unnecessary, also add an `a` in reference to the `backup`, switch `their` to `the`, make `HLQ` plural since datasets is plural, and remove the extra space before the period
- name: Restore data sets from a backup with the original HLQs from when they were archived.